instrumentation/hibernate/hibernate-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/hibernate/SessionMethodUtils.java

```diff
@@ -100,5 +100,12 @@ public static void end(
     targetContextStore.putIfAbsent(target, sessionContext);
   }
 
+  public static String getSessionMethodSpanName(String methodName) {
+    if ("fireLock".equals(methodName)) {
+      methodName = "lock";
```

Comment:
reducing allocation seems worth small code duplication
```suggestion
      return "Session.lock";
```